plenum/server/node.py

```diff
@@ -121,6 +120,25 @@
 logger = getlogger()
 
 
+class GcTimeTracker:
```

Comment:
How this can affect performance? Should we make this metrics disabled by default?

Reply:
Adding callback doesn't measurably affect performance. Asking for gc.get_objects() does add performance spikes once per 10 seconds, but only when there are already lots of objects, in which case we already have much bigger spikes from GC itself, so I don't think this is critical.

Reply:
On a second thought it's not hard to add checks if metrics are enabled, so I'll add them anyways.